run.py

```diff
@@ -527,13 +487,13 @@ def skypemailfix():
 def modbooleanregkeys(regdict):
   
     #This function, when provided with a properly formatted dictionary, wil perform registry changes.
-    # EX; PROPER FORMAT: dictionary = { Title: [HKEY, r'Registry/Path', 'registrykey', keyvalue] }
-    # Title=String, keyvalue must be either 1(Re-enable) or 0(Disable)
+    # EX; PROPER FORMAT: dictionary = { Title: [HKEY, r'Registry/Path', 'registrykey', keytype, keyvalue] }
+    # Title=String, keytype is the type of key such as REG_DWORD, keyvalue must be either 1(Re-enable) or 0(Disable)
     
     for title, registry in regdict.viewitems():
           try:
               modreg = _winreg.OpenKey(_winreg.registry[0], registry[1], 0, _winreg.KEY_ALL_ACCESS)
```

Comment:
Replace _winreg.registry[0] to just registry[0], following the comment that I listed above.